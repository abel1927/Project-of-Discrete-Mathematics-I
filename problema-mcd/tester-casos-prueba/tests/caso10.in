29
61287 71895
505712 571892
621100 945708
377978 720275
447353 794263
172912 876752
302321 600732
355772 815964
468685 544248
43082 1344836
497001 640410
1103790 1162338
235683 698175
729864 905344
68158 131733
730683 1338097
73315 442055
1201035 1245725
672326 861976
634078 1007028
151617 552334
255457 788396
928066 1216931
49918 541672
454259 520738
299722 612482
375654 702374
410749 596873
214786 351619
