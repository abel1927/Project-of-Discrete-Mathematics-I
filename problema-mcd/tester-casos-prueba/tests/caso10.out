19168
142972
157616
563040
789048
53280
172800
134400
181392
316416
85384
193722
2120
51360
69120
1335780
88410
183936
215492
152520
261184
328320
1152540
135416
244080
282672
335896
570900
349920
